<launch>
	<group ns="fmDecisionMakers">
		<node pkg="sdu_pichi_surveyor" name="frobit_remote" type="survey.py" output="screen">
    		<param name="deadman_topic" value="/fmSignals/deadman" />
			<param name="cmd_vel_topic" value="/fmSignals/cmd_vel" />
			<param name="odom_topic" value="/fmKnowledge/odom" />
			<param name="max_linear_velocity" value="0.4"/>
			<param name="max_angular_velocity" value="1.8"/>
			<param name="deadband" value="5"/>
			<param name="reduced_range" value="40"/>
			<param name="feedback_topic" value="/joy/set_feedback" />
			<param name="joy_topic" value="/joy" />
			<param name="status_topic" value="/fmData/status"/>
			<param name="publish_frequency" value="10"/>
			<param name="use_tf" value="False"/>
		</node>
	</group>
        <node pkg="wiimote" type="wiimote_node.py" name="remote_control" respawn="true">
        	<param name="fixed_wiimote_address" value="00:1F:32:B3:A2:F3"/>
		</node>
		
	<group ns="platform_executors">
		<node pkg="position_action_server" name="positionActionServer" type="positionGoalActionServer.py" output="screen">
			<param name="cmd_vel_topic" value="/fmSignals/cmd_vel" />
			<param name="odometry_topic" value="/fmKnowledge/odom" />
			<param name="max_linear_velocity" value="1"/>
			<param name="max_angular_velocity" value="4.0"/>
			<param name="straight_line" value="True"/>
			<param name="straight_line_angle_error" value="0.5"/>
		</node>
	</group>	
</launch>
